components/header/wallet_modal.tsx

```diff
@@ -75,7 +73,7 @@ const WalletModal = ({
 				</ModalButton>
 				{isArconnectAvailable ? (
 					<ModalButton onClick={() => connectToArconnect()}>
-						<Image alt='ArConnect Logo' src={arconnectLogo} width={22} height={22} />
+						<img alt='ArConnect Logo' src='/arconnect.png' width={22} height={22} />
```

Comment:
Why the change from `Image` here? Doesn't next.js provide some nice image scaling for us here?